File: server/src/main/java/by/brawl/ws/service/SpellCastService.java
package by.brawl.ws.service;

import by.brawl.ws.holder.BattlefieldHolder;

public interface SpellCastService {

    BattlefieldHolder castSpell(String id,
                                String senderAccountId,
                                String senderId,
                                Integer target,
                                Boolean enemy,
                                BattlefieldHolder battlefieldHolder);
}

File: server/src/main/java/by/brawl/ws/service/SpellCastService.kt
package by.brawl.ws.service

import by.brawl.ws.holder.BattlefieldHolder

interface SpellCastService {

    fun castSpell(spellId: String,
                  senderId: String,
                  casterId: String,
                  victimPosition: Int?,
                  forEnemy: Boolean?,
                  battlefieldHolder: BattlefieldHolder): BattlefieldHolder

}